bcda/responseutils/writer.go

```diff
@@ -81,7 +81,7 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
 					{
 						Name: "export",
 						Definition: &fhirmodels.Reference{
-							Reference: baseurl + "/api/v1/ExplanationOfBenefit/$export",
+							Reference: baseurl + "/api/v1/Patient/$export?_type=ExplanationOfBenefit",
```

Comment:
Why explicitly indicating the `_type`?  The Capability is the `/Patient` endpoint.  What is the FHIR-compliant way of defining this endpoint in a capability statement.

Reply:
The spec doesn't really provide a "fhir compliant" way of showing this type of endpoint...at least i couldn't find it lol. But i also don't see why we wouldn't want to do this. (It's informing our users how to make the different calls we support via the new query parameters) But it's also possible I'm missing your point entirely. What is it specifically that concerns you??

Reply:
I guess the problem I see is that we should only need to include the endpoint we support (i.e., `/Patient`).  If we start adding all the `_type` options, then it opens up the door to tying us to include all permutations in the CapabilityStatement.